### PR TITLE
Hmtnote

### PR DESCRIPTION
**Pull request by sysbiocoder**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/hmtnote/main.nf [hunk 1]
```diff
@@ -0,0 +1,65 @@
+process HMTNOTE {
+    tag "$meta.id"
+    label 'process_low'
+
+    // TODO nf-core: List required Conda package(s).
+    //               Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    //               For Conda, the build (i.e. "h9402c20_2") must be EXCLUDED to support installation on different operating systems.
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
```
Comment (matthdsm) [inline modules/hmtnote/main.nf:8]: Please remove all comments and TODO's

**Review by matthdsm — CHANGES_REQUESTED** [review]

File: modules/hmtnote/main.nf (hunk 1 above)
Comment (sysbiocoder) [reply modules/hmtnote/main.nf]: Thank you for reviewing, I have done the changes as mentioned

**Review by Emiller88 — APPROVED** [review]
Looks good to me!

**Comment by sysbiocoder** [comment]
@matthdsm  I have made required changes could you please approve if there are no other changes to be done  so that the merging could be done

**Review by matthdsm — APPROVED** [review]

**Pull request merged**